arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/TxInitializationSection.java

```diff
@@ -37,7 +37,9 @@
 
 public class TxInitializationSection extends TraceSection {
 
-  @Getter private final AccountSnapshot senderAfterPayingForTransaction;
+  @Getter private final AccountSnapshot senderAfterPayingForGas;
+  @Getter private final AccountSnapshot senderAfterPayingForValue;
+  @Getter private final AccountSnapshot senderAfterPayingForGasAndValue;
```

Comment:
I would suggest, for the sheer purpose of readability, to have the following snapshots:
```java
private final AccountSnapshot senderGasPayment
private final AccountSnapshot senderGasPaymentNew
private final AccountSnapshot senderValueTransfer
private final AccountSnapshot senderValueTransferNew
private final AccountSnapshot recipientValueReception
private final AccountSnapshot recipientValueReceptionNew
private final AccountSnapshot senderUndoingValueTransfer
private final AccountSnapshot senderUndoingValueTransferNew
private final AccountSnapshot recipientUndoingValueReception
private final AccountSnapshot recipientUndoingValueReceptionNew
```
I've started just introducing such shorthands as it became unmanageable otherwise in other sections (`CallSection` or `CreateSection` for instance)

Reply:
Ok.

Reply:
Do we actually need all of them? `senderGasPaymentNew` and `senderValueTransfer` should be the same.

Reply:
Also, `senderValueTransferNew` and `senderUndoingValueTransfer`, `recipientValueReceptionNew` and `recipientUndoingValueReception` should be the same.

Reply:
Note: whenever variables represent the same thing, create a deep copy of the underlying objects.